### PR TITLE
task 11 complete

### PR DESCRIPTION
**Pull request by ojtaylor123**
Task 11 done on the same branch as task 10 (disregard these tutor comments) in commit "task 11 complete" - task 10 is complete and amendments made for comment counts are made in the latest commit

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -114,6 +114,24 @@ describe("get reviews by id", () => {
...
+      .expect(200)
+      .then(({ body }) => {
+        const { review } = body;
+        expect(Array.isArray(review)).toBe(true);
+        expect(review.length).toBe(1);
+        expect(review[0] && typeof review[0] === "object").toBe(true);
+
+        expect(review[0]).toMatchObject({
+          review_id: 3,
+          comment_count: "3",
```
Comment (Rayhaan2k9) [inline __tests__/app.test.js]: The comment count should be a number

**Review by Rayhaan2k9 — CHANGES_REQUESTED** [review]
Looks great but the comment count should be a number. Once you have fixed this, please resubmit the PR 🙂

File: __tests__/app.test.js (hunk 1 above)
Comment (ojtaylor123) [reply __tests__/app.test.js]: Changes completed in commit 38e076b

**Review by MKD01 — APPROVED** [review]
Following the zoom call happy for this to be merged after those changes have been made

**Pull request merged**